src/components/Tooltip/Tooltip.js

```diff
@@ -0,0 +1,199 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { DialogOverlay, DialogContent } from '@reach/dialog'
+import { useTransition, animated } from 'react-spring'
+import { Manager, Reference, Popper } from 'react-popper'
+
+import { TitleLarge, Body, Footnote } from '../index'
+
+import styles from './Tooltip.module.scss'
+
+const AnimatedModalOverlay = animated(DialogOverlay)
+const AnimatedModalContent = animated(DialogContent)
+
+export const Tooltip = ({ label, placement, details, inline, children }) => {
+  const [tooltipVisible, setTooltipVisibility] = useState(false)
+  const [modalVisible, setModalVisibility] = useState(false)
+
+  const animationConfig = {
+    tension: 250,
+    clamp: true,
+  }
+
+  const transitions = useTransition(modalVisible, null, {
+    config: animationConfig,
+    from: { opacity: 0, transform: 'translate3d(0px, -20px, 0px)' },
```

Comment:
I wonder if the pixel values can be sync'd with our css `--space-N` ones? That would be ideal

Reply:
Synced and created  `--UncommonSpace-N` vars for situations like this

Reply:
Actually wait this is in in the `.js` I am not sure how to do this

Reply:
Removed this transition